.github/workflows/pr.yaml

```diff
@@ -396,27 +413,25 @@ jobs:
       with:
         name: Workflow artifacts
         path: ../workflow_artifacts/
-    - name: Determine latest galaxy commit
-      run: echo ::set-env name=GALAXY_HEAD_SHA::$(cat ../workflow_artifacts/galaxy.sha)
     - name: Cache .cache/pip
       uses: actions/cache@v2
       id: cache-pip
       with:
         path: ~/.cache/pip
-        key: pip_cache_py_${{ matrix.python-version }}_gxy_${{ env.GALAXY_HEAD_SHA }}
+        key: pip_cache_py_${{ matrix.python-version }}_gxy_${{ needs.setup.outputs.galaxy_head_sha }}
     - name: Install Planemo
       run: pip install planemo
     - name: Deploy on testtoolshed
       env:
-        SHED_KEY: ${{ secrets.tts_api_key }}
+        SHED_KEY: ${{ secrets.TTS_API_KEY }}
```

Comment:
@nsoranzo do you know if the change to uppercase requires an update of the settings?

Reply:
Yes, it does need someone with admin privileges on this repo to remove and re-add the TTS and TS API key secrets.

Reply:
OK. Given the activity on this repo we can ask for this change after this is merged.

Reply:
I doubt this is needed, we have uppercase version in the secrets since June

Reply:
I remember that they are shown upper case even if entered lower case. But we can just try with one of my long open PRs :)

Reply:
@bernt-matthias lol, in that case I have no idea what we have. I am ready to change it if it fails though.